### PR TITLE
Merging the branch

### PR DESCRIPTION
**Pull request by KonradSuli**
Fixing 2 sonarqube errors.

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/GT4500.java [hunk 1]
```diff
@@ -20,6 +20,45 @@ public boolean fireLaser(FiringMode firingMode) {
     return false;
   }
 
+  private boolean tryToFirePrimaryFirst() {
```
Comment (FTim) [inline src/main/java/hu/bme/mit/spaceship/GT4500.java]: Still a better solution than mine.

File: src/main/java/hu/bme/mit/spaceship/GT4500.java [hunk 2]
```diff
@@ -20,6 +20,45 @@ public boolean fireLaser(FiringMode firingMode) {
     return false;
   }
```
Comment (FTim) [inline src/main/java/hu/bme/mit/spaceship/GT4500.java]: Write some more comments for the next task. (Because why not.... comments are fiiiine....)

**Review by FTim — CHANGES_REQUESTED** [review]
This is the summary comment.

File: src/main/java/hu/bme/mit/spaceship/GT4500.java (hunk 1 above)
Comment (KonradSuli) [reply src/main/java/hu/bme/mit/spaceship/GT4500.java]: Thanks

**Review by FTim — APPROVED** [review]

**Pull request merged**